src/newshop.cpp

```diff
@@ -1838,8 +1838,11 @@ void shop_list(char *arg, struct char_data *ch, struct char_data *keeper, vnum_t
   if (SHOULD_SEE_TIPS(ch))
     send_to_char("\r\nUse ^WPROBE^n for more details.\r\n", ch);
 
-  if (has_availtns)
+  if (has_availtns) {
     send_to_char(ch, "This shop uses %s for difficult purchases.\r\n", skills[shop_table[shop_nr].etiquette].name);
+    if (SHOULD_SEE_TIPS(ch) && get_metavariant_penalty)
+      send_to_char("\r\n^wMetavariants^n may suffer etiquette/negotiation penalties.\r\n", ch);
```

Comment:
Good idea! Let's change this to something like `<character's race pluralized> have a -4 TN penalty here.`

Reply:
I'm not sure about the singular/plural form of fomori/oni, so I'm now using:
> NOTE: <a/an> <race> will suffer penalties to etiquette/negotiation tests here.

Reply:
IMO they're like sheep, so their singular and plural are the same. That's also written in the helpfiles, like "Oni display" and "Fomori possess", so I think it's a reasonable take on it.

I can write that change if you want to be done with this PR!

Reply:
Yeah, probably better if you take over, since I'm also waffling between making another array like `pc_race_types_decap` for the plural forms, or using a define like `AN`.

Reply:
I'll just make an array for it.